DeepSpeech.py

```diff
@@ -580,9 +580,11 @@ def train():
             loaded = True
 
         if not loaded and FLAGS.load in ['auto', 'last']:
+            tf.initialize_all_variables().run()
```

Comment:
Doing this will hide problems in the checkpoint and continue training from a partially initialized state. Why do you need to run this?